core/common/src/test/java/alluxio/resource/ResourcePoolTest.java

```diff
@@ -30,13 +30,27 @@
   @Rule
   public ExpectedException mThrown = ExpectedException.none();
 
+  /**
+   * Test unit for {@code ResourcePool} class.
```

Comment:
Thanks for your comments and I have updated the javadoc according to your comments.